### ISSUE TITLE
How to interpret the values

### ISSUE THREAD

**Issue by andkov** [dss-hmi/drug-recedivism-2019#1]
@cking34 , I am working on computing various dummies and have stumbled on this example. Person with `id = 35302`, how would we interpret rows 4 and 5?  Are you aware if there is a practice to grant separate convictions with different offense counts?

![image](https://user-images.githubusercontent.com/2608073/57728002-abe24880-7647-11e9-97c1-ab4d09ed3d0a.png)

**Comment by cking34** [comment]
Maybe this person committed multiple burglaries? Not aware of it. I am not sure at this point whether that would affect our analysis or not. I would lean towards not.

**Comment by andkov** [comment]
@cking34 , I am not sure I will be able to finish the EDA-2 today, however, i have made great progress, hitting most of the computation goals we have identified.   I attach a view for three individuals to study the newly created variables. 
![image](https://user-images.githubusercontent.com/2608073/57737899-2fa92e80-7662-11e9-8292-56eb5d94beae.png)

Here is the script behind the computation:
```r
d1 <- ds_sample %>% 
  dplyr::filter(person_id %in% c(46222,65392, 50495) ) %>% 
  dplyr::arrange(person_id, begin_date, offense_arrest_cd) %>% 
  dplyr::group_by(person_id) %>% 
  dplyr::mutate(
    conviction_order = row_number()
  ) %>% 
  dplyr::mutate(
    drug_related = ifelse(offense_group == "C", TRUE, NA)
    ,drug_order = cumsum(!is.na(drug_related))
    ,drug_order = ifelse(is.na(drug_related), NA, drug_order)
    ,after_1996   = ifelse(begin_date > "1996-01-01", TRUE, NA)
    ,days_bw_conv = begin_date - dplyr::lag(begin_date,1)
  )

```
please refer to `./analysis/eda-2/eda-2.R` for the full context.

**Comment by cking34** [comment]
Great. This is starting to look interesting. 

For the graph of offenses over time for those with and without drug felons (I don't think we did that yet), I think that we might need some additional variables using one or more of the three new ones that you created.

I see that you created the drug_related conviction for the offenses that were drug-related. Can you create an additional dummy variable that takes on the value 1 for all the felons who committed a drug felony? For example, this variable would have a "1" for person_id "46222" for ALL the rows because he committed a drug felony? Then for person_id "50495", this variable would have a "0" for ALL the rows because he was never convicted for a drug felony. I think that this variable will allow us to create the 4th graph that we had in EDA-1 for those with and without a drug felony.

Does that make sense?

**Comment by andkov** [comment]
Yes, that makes sense. This should be straightforward. Thanks for the feedback.

**Comment by andkov** [comment]
@cking34, here's the updated view, in response to your comment:
![image](https://user-images.githubusercontent.com/2608073/57820702-b46b7980-7742-11e9-8f7e-6e4f0bbec685.png)

**Comment by andkov** [comment]
It seems that we need another set of variables:
- dummy for whether a person has any convictions after getting a drug-related conviction after 1996

**Comment by cking34** [comment]
OK cool. So it seems that you created the dummy variables that we need. 

I see that you created "drug_ever" which indicates whether the person has ever committed a drug felony. You also created "drug_after"1996" and "drug_before_1996" which should be mutually exclusive. I think these look good. I am not sure whether we might need "n_drug_related", "n_after_1996", and "n_drug_related_after_1996" but these are always good to have. 

About your last comment, yes, I think we need that. It is not very straightforward to create. We need to find whether people have any subsequent conviction if they have drug_after_1996==TRUE

**Comment by cking34** [comment]
There was a National Institute of Justice funding announcement out on Drugs and Crime. The deadline was last week. Too bad. I will keep an eye on other funding opportunities
